### DISCUSSION TITLE
How to show More than one System Context and drilldown

### DISCUSSION THREAD

**Discussion by samuelrdran** [structurizr/lite#79] [Q&A]
Hi there,

How can I enable container level drilldown for more than one systems from the Context level diagram?

Example:
       systemContext salesforce "ContextSalesforce" {
            include * ->anotherSystem1-> 
        }

		container salesforce "ContainerSalesforce" {
			include * 
		}
		
		container anotherSystem1 "ContainerAnotherSystem1" {
			include *
		}

The above does not show the magnify sign for AnotherSystem1 in the Contex view. I tried to add another systemContext view for AnotherSystem1 and that doesn't work either.

Thanks,
Samuel

**Comment by simonbrowndotje** [comment]
It should happen automatically ... please include a full example that can be pasted into https://structurizr.com/dsl so I can help further.
